dask/dataframe/tests/test_dataframe.py

```diff
@@ -6068,25 +6068,37 @@ def test_mask_where_callable():
     assert_eq(pdf.x.where(lambda d: d == 1, 2), ddf.x.where(lambda d: d == 1, 2))
 
 
-@pytest.mark.parametrize("self_destruct", [True, False])
-def test_pyarrow_conversion_dispatch(self_destruct):
```

Comment:
It seems like we should still test  `from_pyarrow_table_dispatch` somewhere. Am I missing this?

Reply:
Good catch, I've messed up copy+pasting here. https://github.com/dask/dask/pull/10501